lib/bindings/HTTPBinding.js

```diff
@@ -121,9 +121,9 @@ function executeCommand(apiKey, device, serializedPayload, callback) {
         options.timeout = config.getConfig().http.timeout;
     }
 
-    request(options, function(error, response, body) {
-        if (error || (response.statusCode !== 200 && response.statusCode !== 201)) {
-            callback(new errors.HTTPCommandResponseError(response.statusCode, error));
+    request(options, function(error, response, body) {        
```

Comment:
Trailing whitespace should be removed.

Reply:
fixed in https://github.com/telefonicaid/iotagent-json/pull/454/commits/e18ea915a788a0b211a0c890113cad3f1e2b7206